client/script/ssCreate.js

```diff
@@ -335,23 +338,31 @@ var submitAll = function (sessionstr, emailstr, targetUrl, inputSources, la) {
     });
   })
   .then(function (response) {
+    var submitTime = new Date();
+    submitEntries.push({"Time": submitTime, "Submitted": true});
     console.log(response);
-    alert("Submitted data.");
+    alertify.alert("<img src='style/accept.png' alt='Success'> Success!", "&nbsp Submitted data.");
```

Comment:
Do we need a non-breakable space here? Why not use CSS for positioning? Otherwise use semicolon at end.